umm-spec-lib/src/cmr/umm_spec/acl_matchers.clj

```diff
@@ -151,13 +152,16 @@
 (defmethod add-acl-enforcement-fields-to-concept :collection
   [concept]
   (-> concept
-      (u/lazy-assoc :AccessConstraints (umm-spec-core/parse-collection-access-value concept))
-      (u/lazy-assoc :TemporalExtents (umm-spec-core/parse-collection-temporal concept))
+      (u/lazy-assoc :AccessConstraints (umm-spec-core/parse-concept-access-value concept))
```

Comment:
Do we have tests for collections?

Reply:
Only for echo10, added tests for other formats.